app/models/presenter/bid.rb

```diff
@@ -81,6 +82,10 @@ def bidder_name
       def amount
         NULL
       end
+
+      def raw_amount
+        nil
```

Comment:
??

Are there nil checks elsewhere that deal with this? If so might be better to find the default value. The goal of a Null object is to be able to answer all the questions you would ask to a normal object and get back reasonable answers.